test/goose/brokers/redis/batch_test.clj

```diff
@@ -58,15 +64,23 @@
 
       (redis-batch/enqueue tu/redis-conn batch)
 
-      (is (= (redis-cmds/parse-map tu/redis-conn batch-state-key) batch-state))
+      (is (= (->> (redis-cmds/parse-map tu/redis-conn batch-state-key)
+               (reduce (fn [m [k v]] (assoc m k (str v))) {}))
```

Comment:
question: we are stringifying the lhs here, and did the same to rhs (batch-state) in the let block above. So why do we need this, cant we remove stringify from both sides?
And, if we do need this, isn't there some clj fn to call to do this directly?

Reply:
From our discussion, 

Redis converts all values to strings and we need to stringify both values so that we can compare them. I've added a comment to clarify this.

I couldn't find a clj fn that does this. Let me know if there is one that I may have missed. 
The reduce itself felt trivial to abstract into a util fn.